src/locales/base-variables-dark.scss

```diff
@@ -0,0 +1,21 @@
+// Options

```

Comment:
Let's not call these "locales" if they are actually "locale + theme". Applies to changes below as well.